cmd/bnscli/clitests/batch.test

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+msgs=`tempfile`
+
+# Combine all messages in a single file to build a batch transaction.
+bnscli send-tokens -src 'seq:foo/src/1' -dst "seq:foo/dst/1" -amount "2 IOV" -memo "sending 2 IOV" >> $msgs
+bnscli send-tokens -src 'seq:foo/src/2' -dst "seq:foo/dst/2" -amount "9 DOGE" -memo "sending 9 DOGE" >> $msgs
+bnscli send-tokens -src 'seq:foo/src/3' -dst "seq:foo/dst/3" -amount "7 BTC" -memo "sending 7 BTC" >> $msgs
+
+bnscli as-batch < $msgs | bnscli view
```

Comment:
How about `rm $msgs` for cleanup at the end? to avoid filling up my /tmp directory?

Reply:
(same for batch_proposal)

Reply:
I can write it instead 

```sh
(
    bnscli send-tokens ...
    bnscli send-tokens ...
    ...
) | bnscli as-batch | bnscli view
```
This way I don't have to use a temporary file. I thought this is less readable so I went with temporary file.
What do you think?